index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "nice-try" {
+  export default function niceTry(f: (...args: any) => any): ReturnType<typeof f>;
```

Comment:
I think it's still wrong because the returned function returns `undefined` if an exception is thrown. Maybe it should be `ReturnType<typeof f> | undefined` or something like that.

Anyways, the better approach is to just not use this module, then you won't need types here at all 😛